doc/cookbook/source/examples/binary_classifier/averaged_perceptron.rst

```diff
@@ -0,0 +1,61 @@
+===================
+Averaged Perceptron
+===================
+
+The Averaged Perceptron is an online binary classifier which is used to determine whether an input,
+namely a feature vector :math:`\mathbf{x}`, belongs to some specific class or not. It is an extension
+of the simple perceptron algorithm because it uses the `average weight` vector
+and the `average bias` to classify the given inputs.
+The predicted class :math:`c` is given by the function below:
+
+.. math::
+
+    c = \theta\left(\mathbf{w} \cdot \mathbf{x}+b\right)
+
+Where :math:`\mathbf{w}` is the average weight vector, :math:`\mathbf{x}` is the input vector we want to classify,
```

Comment:
Grammar error, this is either the same sentence, or a new one (no capital W)

Better:
"Here, w is the average weight vector and b is the average bias. and \theta is a step function: